README.md

```diff
@@ -0,0 +1,43 @@
+# Catalyst
+
+Catalyst is a lightweight, fast and simple to use webserver written on top of hyper.  
+
+## Getting Started
+
+```sh
+$ git clone https://github.com/technetos/catalyst.git
+```
+
```

Comment:
Adding a running the project section would be good. exp:


### Building the project
To build catalyst, simply run
```
$ cargo build
```
For a more performant build,  run with the `--release` flag
```
$ cargo build --release
```

### Running the project
To start Catalyst, run:
```
$ cargo run
```
 then navigate to `http://localhost:8000/foo` and you'll be greeted by the message: `{"message":"Greetings earthling"}`